src/app/static/src/app/store/dataExploration/actions.ts

```diff
@@ -0,0 +1,61 @@
+import {ActionContext, ActionTree} from "vuex";
+import {RootState} from "../../root";
+import {LanguageActions} from "../language/language";
+import {changeLanguage} from "../language/actions";
+import {api} from "../../apiService";
+import {VersionInfo} from "../../generated";
+import {currentHintVersion} from "../../hintVersion";
+import {DataExplorationState} from "./dataExploration";
+import {ErrorReportManualDetails} from "../../types";
+import {LanguageMutation} from "../language/mutations";
+import {DataExplorationMutation} from "./mutations";
+
+export interface DataExplorationActions extends LanguageActions<DataExplorationState> {
+    generateErrorReport: (store: ActionContext<DataExplorationState, DataExplorationState>,
+                          payload: ErrorReportManualDetails) => void;
+}
+
+export const actions: ActionTree<DataExplorationState, DataExplorationState> & DataExplorationActions = {
+
+    async changeLanguage(context, payload) {
```

Comment:
This is subtly different from the root action because the latter references the `modelCalibrate` module and this does not. Possibly there is some nice way to share code, maybe not worth it.